src/octopeer-github/test/content/DOMRewriterTest.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Created by Maarten on 12-06-2016.
```

Comment:
Thanks for adding the tests! :D
The other changes are related to the private and unreachable functions of the controllers, so I'm fine with that. :)